common/libs/wallet/deterministic.js

```diff
@@ -4,17 +4,19 @@ import type { IWallet } from './IWallet';
 export default class DeterministicWallet implements IWallet {
   address: string;
   dPath: string;
+  index: number;
 
-  constructor(address: string, dPath: string) {
+  constructor(address: string, dPath: string, index: number) {
```

Comment:
I chose to make this a separate param, rather than sending it all in as the dPath, to make sure people don't make the same mistake I do!

Reply:
I think thats fine, whoever next works on this should be able to figure out dPath is really dPath - index in that case.